stl/inc/format

```diff
@@ -2557,8 +2557,202 @@ template <_Format_supported_charT _CharT, class _Traits>
 struct formatter<basic_string_view<_CharT, _Traits>, _CharT>
     : _Formatter_base<basic_string_view<_CharT, _Traits>, _CharT, _Basic_format_arg_type::_String_type> {};
 
-using format_context  = basic_format_context<back_insert_iterator<string>, string::value_type>;
-using wformat_context = basic_format_context<back_insert_iterator<wstring>, wstring::value_type>;
+template <class _Ty>
+class _Fmt_buffer {
+private:
+    _Ty* _Ptr        = nullptr;
+    size_t _Size     = 0;
+    size_t _Capacity = 0;
+
+protected:
+    explicit _Fmt_buffer(const size_t _Sz) noexcept : _Size(_Sz), _Capacity(_Sz) {}
+    _Fmt_buffer(_Ty* _Data, const size_t _Sz, const size_t _Cap) noexcept : _Ptr(_Data), _Size(_Sz), _Capacity(_Cap) {}

```

Comment:
All function definitions should be separated with an empty line (exluding `= default` and `= delete`). Ditto below

Reply:
Done